src/doc/templates/templates_term.nit

```diff
@@ -0,0 +1,324 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Markdown templates for Nit model MEntities.
+#
+# This module introduces Markdown rendering services in model entities.
+# With the `no_color` option set to `false`, the output can be highlighted for console.
+module templates_term
+
+import model_collect
+import console
+
+redef class MDoc
+	# Returns the full comment formatted for the console
+	fun cs_comment(no_color: nullable Bool, indent: nullable Int): String do
+		var res = new FlatBuffer
+		indent = indent or else 0
+		for line in content do
+			res.append "{" " * indent}{line}\n"
+		end
+		if no_color == null or not no_color then
```

Comment:
alternative `nullable bool` patterns are:

* `if no_color == true then` but is weird because the nullness is not explicit and that makes the test looks like a truism
* `if no_color or else true then` is nice because we do not repeat the `no_color` and the behavior of the null case is explicit, but a `or else` in a `if` might be confusing to some

Reply:
I used all three alternatives in different projects. I still not sure of my favorite.

So I left it as it is.